File: src/main/java/com/github/urlshorten/UrlShortenApplication.java
package com.github.urlshorten;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@EnableTransactionManagement
@SpringBootApplication
public class UrlShortenApplication {

	public static void main(String[] args) {
		SpringApplication.run(UrlShortenApplication.class, args);
	}

}

File: src/main/kotlin/com/github/urlshorten/UrlShortenApplication.kt
package com.github.urlshorten

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.transaction.annotation.EnableTransactionManagement

@EnableTransactionManagement
@SpringBootApplication
open class UrlShortenApplication

fun main(args: Array<String>) {
    runApplication<UrlShortenApplication>(*args)
}